store/connection.js

```diff
@@ -15,6 +17,24 @@ function onConnectionChange(dispatch, getState, connectionState) {
 
     dispatch(refreshPosts());
   }
+
+  if (
+    connectionState.isConnected &&
+    connectionState.isInternetReachable &&
+    getState().auth.offline
+  ) {
+    // Re-login if the device was offline
+    dispatch(softLogout);
+
+    (async () => {
+      const email = await SecureStore.getItemAsync("CREDENTIALS_EMAIL");
+      const password = await SecureStore.getItemAsync("CREDENTIALS_PASSWORD");
+
+      if (email && password) {
+        await dispatch(login(email, password));
```

Comment:
I think login needs to be imported here

Reply:
Good catch, I just realized that this is actually not necessary at all though